app/views/lists/show.html.erb

```diff
@@ -11,10 +11,13 @@
 </p>
 
 <ul>
-  <% @list.papers.each do |paper| %>
-    <li>
-      <%=paper.title%>
-    </li>
+  <% @list.references.each do |reference| %>
+    <%if reference.paper.present?%>
+      <li>
+      <%=reference.paper.title%> 
+      <%= link_to 'Remove', reference_path(reference) %> 
```

Comment:
It does work and I don't know why.  When I rake routes, I have 2 routes,
one called references_path (which is linked to my create in my controller)
and reference_path (which is my destroy.)  I was confused as well as to why
these got associated this way because I didn't explicitly set this anywhere
that I can tell.


On Sun, Jul 10, 2016 at 12:32 PM, Brian Kung <notifications@github.com>
wrote:

> In app/views/lists/show.html.erb
> <https://github.com/briankung/projectcredo/pull/11#discussion_r70185491>:
>
> > @@ -11,10 +11,13 @@
> >  </p>
> >
> >  <ul>
> > -  <% @list.papers.each do |paper| %>
> > -    <li>
> > -      <%=paper.title%>
> > -    </li>
> > +  <% @list.references.each do |reference| %>
> > +    <%if reference.paper.present?%>
> > +      <li>
> > +      <%=reference.paper.title%>
> > +      <%= link_to 'Remove', reference_path(reference) %>
>
> Does this work? And if so, why does it work? I'd expect you to need to
> send a method: :delete somewhere in there, depending on how the routes
> are wired.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/11/files/67e353f1d94ac8392877ca0f2e76ee0b742ce933#r70185491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACKsZPXu1mjHH_EeyX7bAJtAfTEmy2Irks5qUSzKgaJpZM4JI4Ma>
> .
>